pkg/mapr/DBUtils.go

```diff
@@ -63,8 +62,16 @@ func GetAvailableNodes(clusterID string, operatingSystem string, osVersion strin
 	}
 
 	// query for nodes where the ExpiresAt field has already passed
-	queryStr := fmt.Sprintf(`{"$where":{"$and":[{"$matches":{"NodeObj.OperatingSystem.Name":"(?i)%s"}},{"$lt":{"ExpiresAT": "%s"}},{"$matches":{"NodeObj.OperatingSystem.Version":"%s"}}] }}`,
-		operatingSystem, time.Now().Format(time.RFC3339), osVersion)
+	queryStr := fmt.Sprintf(`
+{"$where":{"$and":[
+	{"$matches":{"NodeObj.OperatingSystem.Name":"(?i)%s"}},
+	{"$lt":{"ExpiresAT": "%s"}},
+	{"$matches":{"NodeObj.OperatingSystem.Version":"%s"}},
+	{"$eq":{"Online": %t}},
+	{"$eq":{"Included":%t}}
```

Comment:
I'm guessing we'll re-do this since we just discussed the idea of removing the "Included" field

Reply:
Done